third_party/forked/godep/pkg.go

```diff
@@ -42,9 +45,19 @@ func LoadPackages(names ...string) (a []*Package, err error) {
 	if len(names) == 0 {
 		return nil, nil
 	}
+
+	pkgs := strings.Split(ignorePackages, ",")
```

Comment:
nit / suggestion: can we call this variable something that makes it clear we want to ignore these?
`LoadPackages` containing a `pkgs` that is not what we're loading is slightly confusing imho.